helpers/wrapper/reports/SECURITY.md

```diff
@@ -1,150 +1,80 @@
-# Security Report: Wrapper
+# Wrapper: Security Report
 
-Smart Contract Security Report by Team Fluidity (team[at]fluidity[dot]io) and Phil Daian (feedback[at]stableset[dot]com)
-Hash of master used for report: [ef5cff0613532d27ecedb332e222ae0a75079841](https://github.com/airswap/airswap-protocols/commit/ef5cff0613532d27ecedb332e222ae0a75079841)
+Security report by Team Fluidity (team[at]fluidity[dot]io). Smart contracts are a nascent space, and no security audit procedure has been perfected. We welcome any suggestions and comments on this report, its contents, our methodology, or potential gaps in coverage.
 
-Wrapper [Source Code](https://github.com/airswap/airswap-protocols/tree/master/helpers/wrapper) and [README](../README.md)
+Wrapper [Source Code](https://github.com/airswap/airswap-protocols/tree/master/helpers/wrapper) and [README](../README.md) are available in this repository. Commit used for report: [2a83c1ff2e46e6befa45889aa556fdd31e5c71fb](https://github.com/airswap/airswap-protocols/commit/2a83c1ff2e46e6befa45889aa556fdd31e5c71fb)
 
 ## Introduction
 
-The Swap Protocol is a peer-to-peer protocol for trading Ethereum tokens that allows two parties to exchange tokens in an atomic transaction. It is a non-custodial exchange settlement contract. The "Wrapper" contract facilitates sending and receiving ether for WETH (wrapped ether) trades on the Swap contract. Contracts are compiled with v0.5.10.a6ea5b19 (0.5.10 stable release).
+Wrapper is a frontend for a Swap contract to enable sending and receiving ether (ETH) for wrapped ether (WETH) trades. The Swap contract only works with tokens, so WETH is required. Wrapper is written and primarily intended for the AirSwap Instant system. If an order does not specify WETH for either party, execution is passed directly through to the Swap contract. The following contracts are compiled with solidity 0.5.12.
 
 ## Structure
 
-Wrapper is comprised of a single contract.
+Wrapper includes one contract and its dependencies.
```

Comment:
`Wrapper includes one contract` I prefer how it used to be - _includes_ is a weird word, it feels like the one contract is included within wrapper, but its not, wrapper IS that contract.